Jenkinsfile

```diff
@@ -1,13 +1,3 @@
-def env_vars = [
-  'SECRET_KEY_BASE=0ae77385a98d4d28886d792832fbbe036152efb4a112fae2d06261850a5b6728',
-  'LINK_HOST=https://www.example.com',
-  'GIBCT_URL=https://www.example.com',
-  'SAML_IDP_METADATA_FILE=.',
-  'SAML_CALLBACK_URL=https://www.example.com/saml/auth/callback',
-  'SAML_IDP_SSO_URL=https://www.example.com/idp/sso',
-  'SAML_ISSUER=GIDS'
-]
-
```

Comment:
I've moved these vars to the docker-compose file as some of them are duplicates and might as well have them in one location. Same with the `withEnv` below.